planb/cli.py

```diff
@@ -78,8 +82,45 @@ def create(regions: list,
     create_cluster(options=locals())
 
 
-sns_topic_help = 'SNS topic name to send Auto-Recovery notifications to'
-sns_email_help = 'Email address to subscribe to Auto-Recovery SNS topic'
+@cli.command()
+@click.option('--region', type=str, required=True)
+@click.option('--cluster-name', type=str, required=True)
+@click.option('--cluster-size', default=3, type=int, help='number of nodes per region, default: 3')
+@click.option('--dc-suffix', type=str, required=True)
```

Comment:
Hm, actually it should be possible to bootstrap a completely new data center from scratch with a non-empty DC suffix.  Then it would make more sense to make this an option in `create` subcommand, instead of having a separate `extend`.  Then it also makes it logical to have `--extend` option in `create`, which will add `auto_bootstrap: false` and `--dc-suffix` won't do that on it's own.